test/CrateIntegrationTest/PDO/PDOStatementTest.php

```diff
@@ -202,8 +202,8 @@ public function testBindNamedParam()
         $name = 'second';
         $id = 2;
         $statement = $this->pdo->prepare('SELECT * FROM test_table where name = :name and id = :id');
-        $statement->bindParam('name', $name);
```

Comment:
Would you prefer a solution that optionally accepts the leading :colon to maintain BC?
I was just trying to implement it to spec http://php.net/manual/en/pdostatement.bindparam.php